### ISSUE TITLE
Using Neutral density vs Potential Density for AABW/CDW definitions

### ISSUE THREAD

**Issue by PaulSpence** [Midway-X/Backward_AABW#4]
Neutral density is annoying (tough to calculate). However, its is believed that the flow in the deep ocean is best aligned with neutral density surfaces (or the the neutral tangent plane). 

http://www.teos-10.org/preteos10_software/neutral_density.html

https://journals.ametsoc.org/view/journals/phoc/27/2/1520-0485_1997_027_0237_andvft_2.0.co_2.xml

Should we be calculating neutral density from the ACCESS-OM2-01 model output and advect Lagrangian parcels through the neutral density vs potential density space?  Is this what are the reviewers from your first paper wanted?

**Comment by Midway-X** [comment]
Thanks for your update. For the last question, I think we only need to calculate the neutral density field at the moment we release to define the seeding range (if we use gamma threshold) and then we only need to calculate the density along particle trajectory to find when particle meets CDW, not the whole ACCESS range.
Since this teos-10 code is only available in MATLAB and Fortran, I think I need to calculate this separately in MATLAB.

**Comment by PaulSpence** [comment]
that makes sense to me.

**Comment by vtamsitt** [comment]
Yes, neutral density is a pain to calculate, and yes, it's a better choice of density and often comes up in review. Not clear to me if it's worth Yinghuan investing in trying to calculate it, but given it's usefulness, maybe calculating it for the ACCESS-OM2-01 output would be useful to a lot of users, not just for this project? I don't know if others have already calculated it for some projects or if there's discussion of including it in future model runs, or if it's just still too time consuming/challenging to compute to be practical.
